P6-Gameplay-Part-2/content.md

```diff
@@ -76,10 +76,8 @@ A great place to add a steady decline would be in the `update(..)` method.
 override func update(currentTime: CFTimeInterval) {
   /* Called before each frame is rendered */
   if state != .Playing { return }
-
```

Comment:
All these new lines are intentional, this is educational source so I prefer to have lots of commented smaller blocks of code.